modules/openapi-generator/src/main/resources/go/client.mustache

```diff
@@ -150,8 +150,12 @@ func parameterValueToString( obj interface{}, key string ) string {
 	return fmt.Sprintf("%v", dataMap[key])
 }
 
-// parameterAddToQuery adds the provided object to the url query supporting deep object syntax
-func parameterAddToQuery(queryParams interface{}, keyPrefix string, obj interface{}, collectionType string) {
+// parameterAddToHeaderOrQuery adds the provided object to the request header or url query
+// supporting deep object syntax
+
```

Comment:
shall we remove these empty blank lines?

Reply:
Oh yea sorry, that's cruft. I'll update now.